cryptography/src/bls12381/mod.rs

```diff
@@ -119,44 +119,11 @@ mod tests {
 
         // Generate the threshold sig
         assert!(matches!(
-            threshold_signature_recover(t, partials).unwrap_err(),
+            threshold_signature_recover(t, &partials).unwrap_err(),
             Error::NotEnoughPartialSignatures(4, 3)
         ));
     }
 
-    #[test]
-    fn test_partial_aggregate_signature_insufficient_duplicates() {
-        let (n, t) = (5, 4);
-        let mut rng = StdRng::seed_from_u64(0);
-
-        // Create the private key polynomial and evaluate it at `n`
-        // points to generate the shares
-        let (group, shares) = generate_shares(&mut rng, None, n, t);
-
-        // Only take t-1 shares
-        let mut shares = shares.into_iter().take(t as usize - 1).collect::<Vec<_>>();
-        shares.push(shares[0]);
```

Comment:
I think it is possible that this test could've passed if we picked a different share to add as a duplicate.